### PR TITLE
Index christian

### PR DESCRIPTION
**Pull request by Teots**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,10 +121,15 @@ <h2>Who are we?</h2>
 
             <ul class="social-links">
               <li><a href="https://www.xing.com/profile/Christian_Stein154"><i class="icon ion-logo-xing social-icon"></i></a></li>
+              <li><a href="https://www.linkedin.com/in/stein-christian"><i class="icon ion-logo-linkedin social-icon"></i></a></li>
+              <li><a href="https://www.github.com/Teots"><i class="icon ion-logo-github social-icon"></i></a></li>
             </ul>
           </cite>
           <br/>
-          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.          
+          First time Christian came into contact with Scala was about 4 years ago when he interview for a position as Data Engineer
```
Comment (pheymann) [inline index.html]: when he interview -> when he interviewed

File: index.html [hunk 2]
```diff
@@ -121,10 +121,15 @@ <h2>Who are we?</h2>
...
+          at the XING SE. Since, then he works on recommender systems, data infrastructure and internal libraries. Especially the
```
Comment (pheymann) [inline index.html]: and -> , and

Comment (Teots) [reply index.html]: Having a comma before `and` is optional, isn't it?

Comment (pheymann) [reply index.html]: Right, it is optional.

Comment (pheymann) [reply index.html]: Then keep it as is.

**Pull request merged**